release-1.8/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** (**GitHub/Slack ID**)  | **Shadow Name(s) (GitHub/Slack ID), ...**
+|  ------ | ------ | ------ |
+|  Lead | | |
+|  Secondary | | |
+|  Features || |
+|  Branch Manager | | |
+|  Test Infra | | | |
+|  Docs || |
+|  Bugs | | | |
+|  CI Signal | | | |
```

Comment:
For 1.8 I think we should merge CI Signal and Upgrade Testing, but the role should start immediately and require keeping the tests passing throughout the release.